test-network/scripts/createChannel.sh

```diff
@@ -19,7 +19,7 @@ fi
 
 createChannelTx() {
```

Comment:
We're no longer creating a channel creation transaction here. Let's rename this to `createChannelGenesisBlock` and update the related comments here and below where we call the function.